common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4198,4 +4203,12 @@ object CelebornConf extends Logging {
       .doc("Interval for refreshing the corresponding dynamic config periodically.")
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("120s")
+
+  val NETWORK_IO_SASL_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.<module>.io.saslTimeout")
+      .categories("network")
+      .doc("Timeout for a single round trip of auth message exchange, in milliseconds.")
+      .version("0.3.2")
```

Comment:
`CELEBORN-1131` only merged into main(0.4.0), so this should be `0.4.0`.

Reply:
done